targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F437xG/TARGET_UBLOX_C030/system_stm32f4xx.c

```diff
@@ -215,10 +214,16 @@ void SystemInit(void)
 #endif /* DATA_IN_ExtSRAM || DATA_IN_ExtSDRAM */
 
   /* Configure the Vector Table location add offset address ------------------*/
+  SCB->VTOR = FLASH_BASE; /* Vector Table Relocation in Internal FLASH */
+
 #ifdef VECT_TAB_SRAM
-  SCB->VTOR = SRAM_BASE | VECT_TAB_OFFSET; /* Vector Table Relocation in Internal SRAM */
-#else
-  SCB->VTOR = FLASH_BASE | VECT_TAB_OFFSET; /* Vector Table Relocation in Internal FLASH */
+  int *vectors = (int *)SCB->VTOR;
```

Comment:
would this result in copying nvic 2x ? I replied the the issue. Why is HalInit part of SystemInit. Can it be delayed and what would be the implications. We are reviewing this currently

cc @c1728p9